packages/perseus/src/widgets/categorizer/categorizer.test.ts

```diff
@@ -145,4 +148,87 @@ describe("categorizer widget", () => {
         // Assert
         expect(userInput).toEqual({values: [1, 0, 0, 0, 1, 1]});
     });
+
+    describe("interactive: full vs answerless", () => {
+        beforeAll(() => {
+            registerAllWidgetsForTesting();
+        });
+
+        let userEvent: UserEvent;
+        beforeEach(() => {
+            userEvent = userEventLib.setup({
+                advanceTimers: jest.advanceTimersByTime,
+            });
+
+            jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
+                testDependencies,
+            );
+        });
+
+        function getAnswerfulItem(): PerseusRenderer {
+            return {
+                content: "[[☃ categorizer 1]]",
+                images: {},
+                widgets: {
+                    "categorizer 1": {
+                        type: "categorizer",
+                        options: {
+                            static: false,
+                            items: ["Circle", "Purple", "Square"],
+                            categories: ["Shape", "Color"],
+                            randomizeItems: false,
+                            values: [0, 1, 0],
+                        },
+                    },
+                },
+            };
+        }
+
+        function getAnswerlessItem(): PerseusRenderer {
+            return splitPerseusItem(getAnswerfulItem());
+        }
+
+        test.each(["answerless", "answerful"])(
+            "is interactive with widget options: %p",
+            async (e) => {
```

Comment:
Why is this parameter named `e`?

Reply:
I followed @handeyeco's  example for this test, so I just kept that the same. Maybe `each`?

Reply:
Changed `e` to `version`. I know that word has a lot of meaning it it, so could also be some other word. `state` also has a lot of meaning. Could be `answerState`?